pages/landing/hipaa/template/pricing/index.html

```diff
@@ -29,24 +29,79 @@
         description: Deploy dev and stage infrastructure today and start building your application. HIPAA compliance coming soon!
       - name: HIPAA-compliant reference architecture
         description: Assessed by qualified 3rd parties for peace of mind.
-      - name: HIPAA-compliant application templates
-        description: World-class application templates that you can easily extend.
+      - name: HIPAA-compliant sample applications
+        description: World-class sample applications that you can easily extend.
       - name: Infrastructure as Code Library
         description: Access our entire library to augment and extend your reference architecture.
       - name: Gruntwork customer support
         description: Gruntwork engineers can answer your technical questions and help you resolve problems if they come up.
-      - name: Access to our library of training videos to ramp up on tools and techniques
+      - name: Access to our library of training videos to ramp up on tools and techniques.
```

Comment:
This is inconsistent with the rest. `name:` has no period, `description` does.

Reply:
Eagle eyes - fixed in dde5ca1a